sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -143,7 +143,8 @@ public SQLDirectExecutor createSQLDirectExecutor(String defaultSchema,
                                                      String preparedStmtName,
                                                      String sqlStmt,
                                                      int defaultLimit) {
-        Session session = createSession(defaultSchema, null, Option.NONE, defaultLimit);
+        User crateUser = userManager.findUser("crate");
```

Comment:
i think it would now make sense to rename this method to something like createSystemExecutor
please also update the docs of this method

if the usermanagement is not enabled it should not set the user.